Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_CrescendoAnole.xml

```diff
@@ -20,7 +20,7 @@
 					<!-- (Note this is a lizard that grows strong when angered) -->
 					<value>
 						<MeleeDodgeChance>0.03</MeleeDodgeChance>
-						<MeleeCritChance>0.25</MeleeCritChance>
+						<MeleeCritChance>0.29</MeleeCritChance>
```

Comment:
That change was unnecessary, I just said in the previous comment that your original values were reasonable. But this is still good. Most good predators have better crit chance.

Reply:
Done(?)